fix: use proper sentence (#651)
harvester/docs: docs/advanced/addons/pcidevices.md

- Hot-plugging and replugging of USB devices is not supported. For more information, see [KubeVirt Issue #11979](https://github.com/kubevirt/kubevirt/issues/11979).

> - If the device path changes when you reattach the device or reboot the node, you must detach it from the virtual machine and follow the steps above to repeat enabling process.

Reviewer: ```suggestion
- If the device path changes when you reattach the device or reboot the node, you must detach the device from the virtual machine and then enable passthrough again.
```

Reviewer: We avoid using words such as "above" and "below" because the relevant information may be relocated in the future.
